### ISSUE TITLE
Is celloracle supporting any gene pertutation beside TF genes?

### ISSUE THREAD

**Issue by bitcometz** [morris-lab/CellOracle#148]
Hello, thanks for this great tool.
I am new here.

Can this software simulate the effect of gene knockout for any gene, apart from the known 2000+ transcription factors?

Thanks！！！

**Comment by yangjun9095** [comment]
Hi @bitcometz ! I'm not the CellOracle developer, but leaving my comment here.

Since CellOracle is designed to model and simulate the in sillico KO based on a "transcriptional" gene regulatory network, I think ti should be transcription factors that are present in your GRN. For example, the effect of in silico K/O is computed by setting the specific TF value to be "0" and computing the change of gene expression of the "target" genes for that TF. So, the perturbation (in silico KO) assumes that you're perturbing the "source" TF in the GRN, whose list depends on your GRN inference steps (cicero to compute the cis-regulatory elements, TSS mapping, and also scanning for TF motifs). 
In other words, it's not designed to simulate translational effectors, cell-signaling molecules, etc., unless they act at the "transcription" level by binding to enhancers. 
Hope this helps!

Best,
Yang-Joon

**Comment by bitcometz** [comment]
thanks for your help !!!
